tests/unit/lms/services/canvas_api_test.py

```diff
@@ -244,6 +244,10 @@ def data_method(self, request, api_client):
 
         return lambda: getattr(api_client, method)(*args)
 
+    @pytest.fixture
+    def api_client(self, pyramid_request):
+        return CanvasAPIClient(sentinel.context, pyramid_request)
```

Comment:
Made a distinction between the public `api_client` and the `base_client`. This should help us separate the tests later on if we split the files up.